x-pack/plugins/infra/public/pages/metrics/metric_detail/asset_detail_page.tsx

```diff
@@ -83,23 +83,6 @@ export const AssetDetailPage = () => {
     [timeRange.from, timeRange.to]
   );
 
-  // Retrocompatibility
-  const handleTabStateChange = useCallback(
-    ({ dateRange: newDateRange }: TabState) => {
-      if (newDateRange) {
-        setTimeRange(
-          {
-            from: newDateRange.from,
-            to: newDateRange.to,
-            interval: timeRange.interval,
-          },
-          false
-        );
-      }
-    },
-    [setTimeRange, timeRange.interval]
-  );
-
```

Comment:
Not sure if this is still needed. Did I miss a use case where this code would be still in use?

Reply:
This is not needed anymore.

Reply:
It's not needed anymore.